Dockerfile

```diff
@@ -6,5 +6,5 @@ COPY environment.yml /
 COPY environment-percolator.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 RUN conda env create -f /environment-percolator.yml && conda clean -a
-ENV PATH /opt/conda/envs/nf-core-mhcquant-1.1.0/bin:$PATH
+ENV PATH /opt/conda/envs/nf-core-mhcquant-1.2.0/bin:$PATH
 ENV PATH /opt/conda/envs/nf-core-mhcquant-percolator-1.0dev/bin:$PATH
```

Comment:
Change this to 1.2.0 too?

Reply:
I was using the nf-core bump-version tool, is there a problem with this?

Reply:
Well, nf-core pipelines should only have a single conda environment. So maybe its time now to consider moving all dependencies in a single environment again? I know its annoying, but we didn't update multiple lines in environment files for that reason ;-) 

Same happens when you use the pipeline with conda only: There is no support for having multiple environment files in Nextflow when using the conda executor IIRC ....

Reply:
Nooo XP, if possible please lets do one more release without having to put them in the same environment...
I have no clue why they dont work together in the same environment, but it might be working with future releases of openms and percolator. However I dont want to wait for this to happen now... as time is running a bit.

Reply:
I guess one more update is fine - but maybe remove the conda profile then, it's not usable anyways...

Reply:
sorry - not sure what you mean by conda profile